corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -41,10 +41,25 @@
 </style>
 <script>
     $(function () {
+        function _analytics(action, callback) {
```

Comment:
A bit confusing that this is only `_analytics` since there is also `analytics.workflow` that is called in other places... Maybe you could call this `_analytics_usage` instead? Otherwise, I'd be inclined to think that this does all things analytics.